Makefile

```diff
@@ -18,9 +20,27 @@ build:
 test:
 	go test ./...
 
+.PHONY: test-with-coverage
+test-with-coverage:
+	go test -cover ./...
+
 .PHONY: cadctl-install-local
 cadctl-install-local: cadctl/cadctl
 
+.PHONY: cadctl-install-local-force
+cadctl-install-local-force: 
+	rm cadctl/cadctl >/dev/null 2>&1  || true
+	make cadctl-install-local
+
+
+# this required-version.go checks that you are at least at go1.16
+# being at go1.16 allows you to go install
+.PHONY: verify-gobin
```

Comment:
If we have this in the go.mod we no longer need this command.

Reply:
We do, that was the solutionfor @ramonbutter's issue